Orm/Xtensive.Orm/Orm/Services/DirectSessionAccessor.cs

```diff
@@ -76,11 +76,11 @@ public IDisposable NullifySessionTransaction()
     /// </summary>
     /// <param name="persistenceState">Type of entity change.</param>
     /// <returns><see cref="EntityState"/>s with the specified <paramref name="persistenceState"/>.</returns>
-    public IEnumerable<EntityState> GetChangedEntities(PersistenceState persistenceState)
-    {
-      return Session.EntityChangeRegistry.GetItems(persistenceState);
-    }
+    public IEnumerable<EntityState> GetChangedEntities(PersistenceState persistenceState) =>
+      Session.EntityChangeRegistry.GetItems(persistenceState);
 
+    internal IEnumerable<EntityState> GetChangedEntitiesInternal(PersistenceState persistenceState) =>
+      Session.EntityChangeRegistry.GetItems(persistenceState);
```

Comment:
What is going on here?

Reply:
My mistake. Fixed to calling internal `EntityChangeRegistry.GetContainer()` to avoid unnecessary `foreach`